README.md

```diff
@@ -97,7 +97,7 @@ network node. It provides the base APIs `config_set`, `config_get`, and
 
 ### CommandReference
 
-The `CommandReference` module provides for the abstraction of NX-OS CLI,
+The `CommandReference` class provides for the abstraction of NX-OS CLI,
```

Comment:
Good thought - more of them will probably be touched by the gRPC work, but this one makes sense to change now. How's the following sound?

> The `CommandReference` class abstracts away the differences between various supported Node types, be that API differences (CLI vs. YANG), platform differences (NX-OS vs. IOS XR), or hardware differences (Nexus 9xxx vs. Nexus 3xxx). A series of YAML files describe various `feature` groupings, with each file describing a set of `attributes` and the specifics of how to inspect and manage that attribute for any supported Node types. When a `Node` is ...